experimental/kubernetes/simple/config/clouddriver.yml

```diff
@@ -74,8 +74,7 @@ dockerRegistry:
       address: ${providers.dockerRegistry.primaryCredentials.address}
       username: ${providers.dockerRegistry.primaryCredentials.username:}
       passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile:}
-      repositories: 
-        - ${providers.dockerRegistry.primaryCredentials.repository}
+      repositories: ${providers.dockerRegistry.primaryCredentials.repositories}
```

Comment:
Spring config can't refere to structs or lists like this, only scalar values & strings.